tests/test_zoom.py

```diff
@@ -14,19 +14,44 @@
 import unittest
 
 import numpy as np
+import torch
 from parameterized import parameterized
 from scipy.ndimage import zoom as zoom_scipy
 
 from monai.data import MetaTensor, set_track_meta
 from monai.transforms import Zoom
-from tests.utils import TEST_NDARRAYS_ALL, NumpyImageTestCase2D, assert_allclose, test_local_inversion
+from monai.transforms.lazy.functional import apply_transforms
+from tests.utils import (
+    DEFAULT_TEST_AFFINE,
+    TEST_NDARRAYS_ALL,
+    NumpyImageTestCase2D,
+    assert_allclose,
+    test_local_inversion,
+)
 
 VALID_CASES = [(1.5, "nearest"), (1.5, "nearest"), (0.8, "bilinear"), (0.8, "area")]
 
 INVALID_CASES = [((None, None), "bilinear", TypeError), ((0.9, 0.9), "s", ValueError)]
 
 
 class TestZoom(NumpyImageTestCase2D):
+    @parameterized.expand(VALID_CASES)
+    def test_pending_ops(self, zoom, mode):
+        im = MetaTensor(self.imt[0], meta={"a": "b", "affine": DEFAULT_TEST_AFFINE})
+        zoom_fn = Zoom(zoom=zoom, mode="bilinear", keep_size=False, dtype=torch.float64)
```

Comment:
I find tests can only passed when dtype is `float64`. The same issue I found in  `Resize` and `Zoom`.

Reply:
that's probably expected, float32 is faster but less accurate